book/website/reproducible-research/rdm/rdm-elns.md

```diff
@@ -0,0 +1,261 @@
+Open formats can have multiple different implementations of tools which read them.
+If there is a good enough specification then new tools can even be written for the format if none of the originals are available anymore.
+Proprietary tools way require some kind of license activation process to use this may not be possible if their vendor no longer exists.
+Using a proprietary solution for archival purposes is therefore taking a needless risk with the future of your data.
+Your data's fate is tied to the format in which it is stored choose it wisely.
+
+(rr-rdm-elns-choosing-lockin)=
+### Lock-in
+
+Most proprietary or open tools will permit you to export your data in one format or another.
+The quality of this export is critical to scrutinise.
+The ability to save all your lab books as PDFs is fine but if it is the only option you may lose a lot of metadata, and the ability to import your data into a new solution. 
+The ability to export your data and retain the original structure is very important to evaluate.
+
+Proprietary solutions are incentivised to attempt to lock-in their customers so that the cost of switching to a competitor is high - so be especially wary of poor export options.
+Note that this tends to change over time. 
+New providers are focused on user acquisition so data portability is often good at first but tends to decline after a certain point in time.
+Firms tends to follow a pattern of shifting from a user acquisition phase with favourable terms, to a user retention phase with less favourable ones.
+Mendeley for example began [encrypting their local database](https://www.zotero.org/support/kb/mendeley_import#mendeley_database_encryption). 
+This made it impossible to migrate your local library to different reference managers without going through the Mendeley online library feature.
+This occurred after they had reached a high degree of market penetration and had been acquired by Elsevier.
+
+When looking for any critical piece of software for long term use the first questions to ask are:
+- Is there an Libre / open-source solution to this problem?
+- If it is a web app and/or has a server: Could I host my own fully featured instance, should I need to?
+- Is there a large/active community using the project, does it have institutional backing of some kind? 
+  This might take the form of a company which sells service contracts, or offers paid hosting ideally with feature parity with a self-hosted option. 
+  Or perhaps a foundation or other non-profit/academic organisation with robust funding.
+- What are the data export options and formats?
+
+Open solutions provide you with the assurance that if you do the appropriate preparatory work you should be able to access all of your data in its native form.
+For example keeping copies of the ELN software that can be run offline in a VM or similar computational environment in the future.
```

Comment:
```suggestion
For example keeping copies of the ELN software that can be run offline in a virtual machine (VM) or similar computational environment in the future.
```
Or add it to the glossary as a definition? The abbreviation hasn't been used before so should be explained in some way